app/code/core/Mage/Dataflow/Model/Profile.php

```diff
@@ -86,7 +86,7 @@ protected function _beforeSave()
     {
         parent::_beforeSave();
         $actionsXML = $this->getData('actions_xml');
-        if (strlen($actionsXML) < 0 &&
+        if ($actionsXML !== null && strlen($actionsXML) < 0 &&
```

Comment:
```suggestion
        // @phpstan-ignore-next-line because of https://github.com/phpstan/phpstan/issues/10570
        if ($actionsXML !== null && strlen($actionsXML) < 0 &&
```